### PR TITLE
Feature/m4928

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import BootstrapVue from 'bootstrap-vue/dist/bootstrap-vue.esm'
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js]: I would move all the CSS to the entry point of your app => main.js

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      state.aggs = aggs
+    },
+    setBiobanks: function (state, items) {
+      state.biobanks = items
+    },
+    setAttributeCharts: function (state, charts) {
+      state.attributeCharts = charts
+    }
+  },
+  actions: {
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js]: It is worthwhile to start moving mutations and actions to their own mutation.js and actions.js files if the number of methods keeps growing

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/main.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// The Vue build version to load with the `import` command
+// (runtime-only or standalone) has been set in webpack.base.conf with an alias.
+import Vue from 'vue'
+import App from './App'
+import store from './store/index'
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/main.js]: import store from './store' will also work

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            title: 'Biobank'
+          },
+          height: 500
+        }
+      }
+    },
+    computed: {
+      ...mapState(['aggs'])
+    },
+    name: 'graph'
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue]: My personal preference, but I always place the name at the top of the component, so its easy to see what to use in the template

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  import Vue from 'vue'
+  import VueCharts from 'vue-charts'
+
+  Vue.use(VueCharts)
+
+  export default {
+    data: function () {
+      const self = this
+      return {
+        chartEvents: {
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue]: Ehh.. a function in data, if you place it in methods() you can call it from the template in the same way

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Filters.vue [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    </select>
+  </form>
+</template>
+
+<script>
+  import FilterCheckbox from './FilterCheckbox'
+  import {mapState} from 'vuex'
+
+  export default {
+    self: this,
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Filters.vue]: self: this is really Reacty, besides you don't use self here

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Filters.vue [hunk 7]
```diff
@@ -0,0 +1,60 @@
+<template>
+  <form>
+    <h1>Filter</h1>
+    <h4>Data types</h4>
+    <div class="custom-controls-stacked">
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Filters.vue]: Don't forget you have the awesome power of bootstrap4 containers, containers-fluid etc..

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/AttributeGraphs.vue [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  </div>
+</template>
+
+<script>
+  import {mapState} from 'vuex'
+
+  export default {
+    data: function () {
+      return {
+        chartEvents: {
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/AttributeGraphs.vue]: Move functions to methods!

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/App.vue [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+  name: 'app',
+  components: {
+    Filters,
+    Graph,
+    AttributeGraphs
+  }
+}
+</script>
+
+<style>
```
Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/App.vue]: remove empty style tags

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js (hunk 1 above)
Comment (dennishendriksen) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js]: Goal of M4928 is a demo with working happy flow with minimum development effort. Code review is always good, but not required in this case.

Comment (Mark-de-Haan) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js]: Thats why the review is a comment and not a request for changes

Comment (fdlk) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js]: I'm quite happy to discuss these in the light of M4822

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue (hunk 5 above)
Comment (fdlk) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue]: Nice, I didn't know that!

File: molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue (hunk 4 above)
Comment (fdlk) [inline molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Graph.vue]: Excellent suggestion, done!

**Pull request merged**